sumologic/sumologic_sources.go

```diff
@@ -30,6 +30,8 @@ type Source struct {
 	Fields                     map[string]interface{} `json:"fields,omitempty"`
 	Url                        string                 `json:"url,omitempty"`
 	ContentType                string                 `json:"contentType,omitempty"`
+	PathExpression             string                 `json:"pathExpression,omitempty"`
+	Encoding                   string                 `json:"encoding,omitempty"`
```

Comment:
missing `deny_list`

Reply:
I agree, I did a pass at trying to get deny_list setup for feature completion (we don't use it for our use case) But I ran into trouble with what data struct is the correct one to be used. Does anyone have a recommendation on how to implement the list? An example from another part of the codebase would also be helpful.

Reply:
Actually I might be wrong, since path_expression, encoding and deny_list are specific to the local file source, they should not be added to the `Source` struct. @vsinghal13 can you confirm?

Reply:
correct, since they are specific to the local file source, these properties should only be added to the struct in `sumologic_local_file_source.go` file.